src/docker.test.ts

```diff
@@ -46,6 +46,7 @@ describe("Docker images", (): void => {
 
```

Comment:
Update commit message to reflect the fact that GitHub Actions pre-caches a standard set of Docker images but that those aren't accessible when using rootless-docker. Also, "Docker" should be capitalized when it's a proper noun.